.azure/cve-pipeline.yaml

```diff
@@ -0,0 +1,89 @@
+# Triggers
+trigger: none
+pr: none
+
+# Parameters
+parameters:
+  - name: releaseVersion
+    displayName: Release Version
+    type: string
+  - name: useSuffix
+    displayName: Build suffixed images
+    type: boolean
+    default: true
+  - name: releaseSuffix
+    displayName: Release Suffix
+    type: number
+  - name: sourcePipelineId
+    displayName: Pipeline ID of the source build
+    type: number
+    default: 16
+    values:
+      - 16
+  - name: sourceBuildId
+    displayName: Build ID of the source build
+    type: number
+
+# Stages
+stages:
+  - stage: container_build
+    displayName: Prepare Container
+    jobs:
+      - template: 'templates/jobs/build/build_containers.yaml'
+        parameters:
+          artifactSource: 'specific'
+          artifactProject: 'strimzi'
+          artifactPipeline: '${{ parameters.sourcePipelineId }}'
+          artifactRunVersion: 'specific'
+          artifactRunId: '${{ parameters.sourceBuildId }}'
+          architectures: ['amd64', 'arm64']
+  - stage: containers_publish_with_suffix
+    displayName: Publish Containers for ${{ parameters.releaseVersion }}-${{ parameters.releaseSuffix }}
+    dependsOn:
+      - container_build
+    condition: and(succeeded(), startsWith(variables['build.sourceBranch'], 'refs/heads/release-'))
+    jobs:
+      - template: 'templates/jobs/build/push_containers.yaml'
+        parameters:
+          dockerTag: '${{ parameters.releaseVersion }}-${{ parameters.releaseSuffix }}'
+          artifactSource: 'current'
+          artifactProject: 'strimzi'
+          artifactPipeline: ''
+          artifactRunVersion: ''
+          artifactRunId: ''
+          architectures: ['amd64', 'arm64']
+  - stage: manual_validation
+    displayName: Validate container before pushing container as ${{ parameters.releaseVersion }}
```

Comment:
I don't remember we have this manual validation in Drain Cleaner and canary for example. Why do we need it here? Is that anything that we should think of the other repo as well?